docs/resources/rediscloud_subscription.md

```diff
@@ -93,16 +94,16 @@ only with Redis Labs internal cloud account
 
 The `creation_plan` block supports:
 
-* `memory_limit_in_gb` - (Required) Maximum memory usage for the initial databases
+* `memory_limit_in_gb` - (Required) Maximum memory usage that will be used your databases.
```

Comment:
```suggestion
* `memory_limit_in_gb` - (Required) Maximum memory usage that will be used for your largest planned database.
```

Reply:
Would this be a better approach? As the current wording could be interpreted to mean "the maximum memory usage of the initial databases _combined_". Could do with some discussion.